vision_evaluation/evaluators.py

```diff
@@ -1044,5 +1175,73 @@ def add_predictions(self, predictions, targets):
         for pred_mask, gt_mask in zip(predictions, targets):
             pred_mask = np.asarray(pred_mask)
             gt_mask = np.asarray(gt_mask)
-            mean_l1 = np.abs(pred_mask.astype(np.float)-gt_mask.astype(np.float)).mean()
+            mean_l1 = np.abs(
+                pred_mask.astype(np.float) - gt_mask.astype(np.float)).mean()
             self._metric_sum += mean_l1
+
+
+class GroupWiseEvaluator(Evaluator):
+    """
+    Group Wise Evaluator
```

Comment:
It would be better if there is more detailed description or a small example :)

Reply:
Added a small descriptive example. Please see if it helps.